.github/workflows/benchmarks.yml

```diff
@@ -72,6 +84,43 @@ jobs:
           pr-bench-name: this-change
           base-bench-name: tip-of-tree
 
+      # lit-html:repeat
+      - name: Benchmark lit-html/repeat
+        run: |
+          cd packages/benchmarks
+          npx tachometer \
+            --config lit-html/repeat/tachometer.json \
+            --json-file lit-html/repeat/results.json
+
+      - name: Report lit-html-repeat
+        uses: andrewiggins/tachometer-reporter-action@v2
+        with:
+          report-id: lit-html-repeat
+          path: packages/benchmarks/lit-html/repeat/results.json
+          pr-bench-name: this-change
+          base-bench-name: tip-of-tree
+
+      # TODO(kschaaf): re-enable this once we work out a good way to trigger
+      # optional benchmarks for a PR
+      # # lit-html:repeat-full
+      # - name: Benchmark lit-html/repeat-full
+      #   if: ${{ contains(github.event.pull_request.body, '[benchmarks:lit-html-repeat-full]') }}
```

Comment:
My thought was that non-repeat specific changes to lit-html core might also affect `repeat` performance, so better to make it a tag you could add to any PR.

The idea was that _if_ you noticed repeat perf changing, you could add the tag to the description (not unlike how you trigger a lot of presubmit features in critique) to trigger the expanded benchmarks.

Either way, this didn't work out so hot, because edits to the metadata (and even manually re-running the workflow after that) don't cause this condition to be evaluated against the changed metadata (title or body); it's only after you push a new commit that the conditions changed, from what I empirically found, making it sort of finicky.

Reply:
OK.